### PR TITLE
Alchemical analysis tool added

### PR DESCRIPTION
**Pull request by tsenapathi**

### CODE REVIEW

File: cheminformatics.yaml [hunk 1]
```diff
@@ -76,6 +76,10 @@ tools:
   - name: gmx_fep
     owner: chemteam
     tool_panel_section_label: ChemicalToolBox
+    
+  - name: alchemical_analysis
```
Comment (bgruening) [inline cheminformatics.yaml]: @tsenapathi this is a suite right? Can you add instead the single tools please.

**Comment by tsenapathi** [comment]
@bgruening hope this is correct

**Comment by bgruening** [comment]
Mh, now you removed a tool ... What do you want to do? Didn't you want to add all 3 tools that you merged yesterday?

**Comment by tsenapathi** [comment]
@bgruening I tried to add the free energy tools as a suite as you suggested. Both gmx_fep tool and alchemical analysis tools are part of the free energy tool suite. How can I do this correctly?

**Comment by chrisbarnettster** [comment]
Hi @bgruening it is just the two free energy tools. Two other recent PR's in compchem were a correction and a tool added by @simonbray that is not for free energy. 
@tsenapathi  please change from a suite to 2 tools as suggested and then all looks good to me.

**Comment by tsenapathi** [comment]
@bgruening @chrisbarnettster apologies for my mistake. Added the two tools gmx_fep and alchemical_analysis as two tools

**Comment by chrisbarnettster** [comment]
@tsenapathi I have installed on my side and tools do not appear in the tool panel. 
Looks like this is an error in directory layout.

**Comment by chrisbarnettster** [comment]
P.S. Installed as a test and tool looks good. Ignore previous (deleted) comment.

**Pull request merged**